### PR TITLE
[CELEBORN-1593][FOLLOWUP] Add CLI support for some missing HTTP APIs

### PR DESCRIPTION
**Pull request by akpatnam25**
### What changes were proposed in this pull request?
Adding support for `removeWorkersUnavailabeInfo`, `getDecommissioningWorkers` and `runIsDecommissioning`. This was missed in the last PR. 

### Why are the changes needed?
see above

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
added unit tests

### CODE REVIEW

**Comment by akpatnam25** [comment]
ping @SteNicholas @turboFei @FMX

**Review by SteNicholas — APPROVED** [review]
LGTM. @akpatnam25, btw, do all REST API have the corresponding cli command? It's better to support other missing command in this pull request.

**Comment by akpatnam25** [comment]
@waitinfuture I think it seems to be taking my work email, which I am not allowed to add as an email. From now on before committing, I can run: `git config user.email` with my verified email address.  I amended the previous commits in this PR to be the correct email now: 
```
apatnam-mn3092:incubator-celeborn apatnam$ git log
commit 57f1c6bad035f7a2dd334d83c745c507e6ae1054 (HEAD -> CELEBORN-1593, origin/CELEBORN-1593)
Author: Aravind Patnam <akpatnam25@gmail.com>
Date:   Thu Sep 12 00:32:26 2024 -0700

    spotless

commit f17c06691fee76ab5766f40222c8e935ba866434
Author: Aravind Patnam <akpatnam25@gmail.com>
Date:   Thu Sep 12 00:27:04 2024 -0700

    [CELEBORN-1593][FOLLOWUP] Add CLI support for removeWorkersUnavailableInfo

```

**Comment by akpatnam25** [comment]
> LGTM. @akpatnam25, btw, do all REST API have the corresponding cli command? It's better to support other missing command in this pull request.

@SteNicholas yes, this was the only one I missed since this API was added a bit later.

File: cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala [hunk 1]
```diff
@@ -77,6 +78,14 @@ class MasterSubcommandImpl extends Runnable with MasterSubcommand {
     workerApi.excludeWorker(removeExcludeWorkerRequest)
   }
 
+  private[master] def runRemoveWorkersUnavailableInfo: HandleResponse = {
+    val workerIds = getWorkerIds
+    val removeWorkersUnavailableInfoRequest =
+      new RemoveWorkersUnavailableInfoRequest().workers(workerIds)
+    logInfo(s"Sending remove workers unavailable info requests to workers: $workerIds")
```
Comment (RexXiong) [inline cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala:85]:
```suggestion
    logInfo(s"Sending remove workers unavailable info requests to master: $workerIds")
```

**Comment by akpatnam25** [comment]
> LGTM. @akpatnam25, btw, do all REST API have the corresponding cli command? It's better to support other missing command in this pull request.

the missing APIs should be updated now in this PR.

**Review by turboFei — APPROVED** [review]

**Comment by akpatnam25** [comment]
ping @SteNicholas @RexXiong @FMX, thanks :)

File: LICENSE-binary [hunk 2]
```diff
@@ -309,6 +309,7 @@ org.slf4j:jcl-over-slf4j
 org.webjars:swagger-ui
 org.xerial.snappy:snappy-java
 org.yaml:snakeyaml
+info.picocli:picocli
```
Comment (FMX) [inline LICENSE-binary:312]: Please keep this license files sorted in alphabetical order.

**Review by FMX — APPROVED** [review]
LGTM except a nit.

File: LICENSE-binary (hunk 2 above)
Comment (akpatnam25) [reply LICENSE-binary]: sure updated

**Comment by akpatnam25** [comment]
thanks, updated @FMX

**Comment by FMX** [comment]
Thanks. Merged into main(v0.6.0).

**Pull request closed without merging**